### ISSUE TITLE
Were you able to find a solution?

### ISSUE THREAD

**Issue by mohdrasbi** [rib/grpc-unity-deadlock-reproduction#1]
Were you able to find a solution?

**Comment by rib** [comment]
No, unfortunately for me I totally wasted my time making a reproducible test case - the maintainers of Google's grpc bindings for C# never looked at the issue or gave any feedback. It was a pretty clear signal to me that upstream has zero interest in maintaining support for Unity and I lost all respect for the project / maintainers considering that they couldn't even muster the energy to respond and say, 'sorry we don't support Unity'.

It looks like one of those Google projects that only really cares about internal priorities, with a pretty shallow attempt at having 'community'. For me I consider that too much of a liability compared to working with projects that have active maintainers that I can easily talk to when I find bugs/issues that need investigating.

So as to not be burned twice I've since made sure to go out of my way to avoid using Google's GRPC project. In general I switched to using msgpack instead of protobuf/grpc but in case I ever need to use GRPC I plan to use the Rust Tonic implementation (since I have a Rust based native plugin for my project anyway).
